nifi-mock/src/main/java/org/apache/nifi/util/MockProcessSession.java

```diff
@@ -753,23 +763,8 @@ public void transfer(final Collection<FlowFile> flowFiles, final Relationship re
             transfer(flowFiles);
             return;
         }
-        if(!processor.getRelationships().contains(relationship)){
```

Comment:
I couldn't see why/where this section of code needs to be remove, can you elaborate?